src/app/ui/icons/icon.tsx

```diff
@@ -0,0 +1,33 @@
+import { styled } from 'leather-styles/jsx';
+import type { HTMLStyledProps } from 'leather-styles/types';
+
+import { HasChildren } from '@app/common/has-children';
+
+// These are needed bc we use a path element in the styled svg
+// which causes type errors if removed
+export interface IconProps extends HTMLStyledProps<'svg'> {
+  color?: string;
+  opacity?: string;
+  strokeWidth?: string;
+  width?: string;
```

Comment:
Saw you replied to comment, but these are still here? Are they needed for some reason?

Reply:
There is a comment in the code left as to why. I removed them, but got type errors bc the underlying component we use is a `path` element not a styled component which seem to create type errors unless I redefined here. I can try using a `styled.path` and removing them but not sure if that will work and did not seem to allow for changing the color with the color picker in storybook (which maybe isn't nec?).

Reply:
I figured out a better way to handle the types, so I was able to remove these again.

Reply:
I think it could be good to keep the `mdx` right beside the story so `thing.stories.mdx` isn't a bad pattern. 

I'm fine with a `docs/` folder too though